lib/jackrabbit.js

```diff
@@ -107,7 +107,7 @@ const jackrabbit = (url, logger, options = {}) => {
 
     const isReconnectionError = (err) => {
 
-        return err.code === 320 || err.message === 'Socket closed abruptly during opening handshake' || err.message.includes('ECONNREFUSED');
+        return err.code === 320 || err.message === 'Socket closed abruptly during opening handshake' || err.message.includes('ECONN');
```

Comment:
Should the error start with `ECONN`?
```suggestion
        return err.code === 320 || err.message === 'Socket closed abruptly during opening handshake' || err.message.startsWith('ECONN');
```

Reply:
No, the error's code could start with `ECONN`, but I'm unsure if that will apply for all connection errors.

As you can see on the test cases, the error messages I've identified so far contains `ECONNRESET` or `ECONNREFUSED`, but don't start with this string (there are even multiple messages which can cause an `ECONNRESET`)